src/operations/booleans/to3DWalls.js

```diff
@@ -0,0 +1,24 @@
+const {vec3} = require('../../math')
+
+const {geom2, geom3, poly3} = require('../../geometry')
+
+const to3DWall = (z0, z1, side) => {
+  const points = [
+    vec3.fromVec2(side[0], z0),
+    vec3.fromVec2(side[1], z0),
+    vec3.fromVec2(side[1], z1),
+    vec3.fromVec2(side[0], z1)
+  ]
+  return poly3.fromPoints(points)
+}
+
+const to3DWalls = (options, geometry) => {
```

Comment:
could you please add a small docstring to make it clearer what this does ? (still quite confusing when you do not know the innards :)

Reply:
Also some thoughts: is a 3D 'wall' like this not just the same as the OUTLINE of a 2D shape, with Z0, Z1 set to the given values ?
meh, overthinking

Reply:
i added documentation. done.